### PR TITLE
Test setup for PostgreSQL 9, 11, 12 and MariaDB

### PR DESCRIPTION
**Pull request by pimeys**

### CODE REVIEW

File: query-engine/connector-test-kit/src/test/scala/writes/nonEmbedded/nestedMutations/NestedDeleteMutationInsideUpdateSpec.scala [hunk 1]
```diff
@@ -1534,7 +1534,7 @@ class NestedDeleteMutationInsideUpdateSpec extends FlatSpec with Matchers with A
          |   }
          |  ) {
          |    nameTop
-         |    middles {
+         |    middles (orderBy: id_ASC) {
```
Comment (janpio) [inline query-engine/connector-test-kit/src/test/scala/writes/nonEmbedded/nestedMutations/NestedDeleteMutationInsideUpdateSpec.scala]: unrelated change?

Comment (pimeys) [reply query-engine/connector-test-kit/src/test/scala/writes/nonEmbedded/nestedMutations/NestedDeleteMutationInsideUpdateSpec.scala]: No. Very much related.

Comment (janpio) [reply query-engine/connector-test-kit/src/test/scala/writes/nonEmbedded/nestedMutations/NestedDeleteMutationInsideUpdateSpec.scala]: care to explain for me?

**Comment by pimeys** [comment]
When @do4gr removed the automatic ordering, he needed to force it in some of the tests to be able to somehow test it with all the connectors. Not all tests needed this then, but now when testing with pg11 and pg12, without explicit ordering the results come in a different order, therefore adding the orderBy to the tests.

-------- Original Message --------
